manifests/script-lab-edge.xml

```diff
@@ -17,7 +17,11 @@
     <AppDomain>https://github.com</AppDomain>
     <AppDomain>https://stackoverflow.com</AppDomain>
     <AppDomain>https://dev.office.com</AppDomain>
-    <AppDomain>https://bornholm-runner-edge.azurewebsites.net</AppDomain>  
+    <AppDomain>https://script-lab.azureedge.net</AppDomain>
+    <AppDomain>https://script-lab-runner.azureedge.net</AppDomain>
+    <AppDomain>https://bornholm-runner-edge.azurewebsites.net</AppDomain>
```

Comment:
Could you order these in some order like production => insiders => edge?
I would also be OK with all environments having all the same app domains list.  Yeah, it'd be slightly overkill, but it's easier to copy-paste, and it wouldn't harm anything to have an extra app domain (even if it's already implied based on where the page is running from)